pyhocon/config_tree.py

```diff
@@ -281,8 +281,11 @@ def get_bool(self, key, default=UndefinedKey):
             'true': True, 'yes': True, 'on': True,
             'false': False, 'no': False, 'off': False
         }
+        string_value = self.get_string(key, default)
+        if string_value is not None:
```

Comment:
Now that `get_string` is not messing with string values, we have to call `lower` ourselves.